lesson12/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="reset.css" type="text/css" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <script src="script.js" defer></script>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div id="js-loading">
+    </div>
```

Comment:
IMO
I think this should be created in JS.
This is because `removeLoadingImg ` removes this entire element.
If you ever reuse `showLoadingImg()`, for example, this would be an error.

Reply:
Thank you for your comment!
It is certainly so.

How about adding an ID "loading-img" to img with showloading () and deleting it with removeloading()?
For example:
````
function showLoadingImg(){
    const loadingImg = document.createElement('img');
    loadingImg.id = "loading-img";
    loadingImg.src = "loading-circle.gif";
    loadingImg.alt = "ローディング画像";
    document.getElementById('js-loading').appendChild(loadingImg);
}

function removeLoadingImg(){
    document.getElementById('loading-img').remove();
}
````

Reply:
Given the increasing complexity of future implementations
It seems to me that there is little advantage in leaving the empty tag with the style attached.

IMO, so I'll leave the fix to you.

Reply:
I fixed https://github.com/nakagawamai/js-lesson/pull/14/commits/b4437b525573a679a83ffff244618a064a9bac5d